docs/topics/pages.rst

```diff
@@ -440,26 +440,25 @@ This is because ``Page`` enforces ordering QuerySets by path. Instead you must a
 
     news_items = NewsItemPage.objects.live().order_by('-publication_date')
 
-Page custom managers
+Custom Page managers
 --------------------
 
 ``Page`` enforces its own 'objects' manager in its ``__init__`` method, so you cannot add a custom manager at the 'objects' attribute.
```

Comment:
I assume this sentence should be removed now?

I'm also not clear on why we need two bits of example code here, although that might just be me being rusty on this area of Django - it looks like we've changed from "you can't do X, but you can do Y instead" to "you can do X, and you can also do Y" with no explanation of the difference between the two or how they differ from standard Django behaviour. (Is it specifically the fact that they should inherit from PageManager? If so, we should highlight that in the text.) My impression is that managers and querysets are pretty much interchangeable now that Manager.from_queryset exists, so perhaps we can get away with a single example.

Filtering EventPage to just future events strikes me as a slightly dubious thing to do within the *default* manager - perhaps the example(s) could demonstrate adding a `.future()` method to the manager/queryset instead?